tests/csapi/sync_test.go

```diff
@@ -382,6 +382,50 @@ func TestSync(t *testing.T) {
 	})
 }
 
+// Test presence from people in 2 different rooms in incremental sync
+func TestPresenceSyncDifferentRooms(t *testing.T) {
+	runtime.SkipIf(t, runtime.Dendrite) // does not yet pass
```

Comment:
I would appreciate it if you could change this to a concrete `FIXME:` comment, as I'm currently changing the whole codebase over as of https://github.com/matrix-org/complement/pull/519.

I think this test would be affected by https://github.com/matrix-org/dendrite/issues/2803, but please do due diligence next time figuring out what makes this test fail, or at least make an issue here noting it.

Reply:
It's not working, I've added a FIXME referencing your bug since chances that this is related are high indeed.

Reply:
I think adding this FIXME is alright for now, when that issue is fixed, i'm sure to find this test to un-skip.